digiroad2-oracle/src/main/resources/db/migration/V236_create_new_roawday_asset.sql

```diff
@@ -0,0 +1,441 @@
+-- NEW ASSET TYPE
+INSERT INTO ASSET_TYPE (ID, NAME, GEOMETRY_TYPE, CREATED_BY)
+VALUES (450, 'K1 keskiviiva', 'linear', 'db_migration_v236');
+INSERT INTO ASSET_TYPE (ID, NAME, GEOMETRY_TYPE, CREATED_BY)
+VALUES (460, 'K2 ajokaistaviiva', 'linear', 'db_migration_v236');
+INSERT INTO ASSET_TYPE (ID, NAME, GEOMETRY_TYPE, CREATED_BY)
+VALUES (470, 'K3 Sulkuviiva', 'linear', 'db_migration_v236');
+INSERT INTO ASSET_TYPE (ID, NAME, GEOMETRY_TYPE, CREATED_BY)
+VALUES (480, 'K4 Varoitusviiva', 'linear', 'db_migration_v236');
+INSERT INTO ASSET_TYPE (ID, NAME, GEOMETRY_TYPE, CREATED_BY)
+VALUES (490, 'K5 Sulkualue', 'linear', 'db_migration_v236');
+INSERT INTO ASSET_TYPE (ID, NAME, GEOMETRY_TYPE, CREATED_BY)
+VALUES (500, 'K6 Reunaviiva', 'linear', 'db_migration_v236');
```

Comment:
These values are not an asset type, they are values of one field (properties).
Suggestion
```SQL
-- ASSET TYPE VALUE
INSERT INTO ASSET_TYPE (ID, NAME, GEOMETRY_TYPE, CREATED_BY)
    VALUES ( 460, 'Name of the Asset in Finnish', 'linear', 'db_migration_v236');